Tests/PersistenceHandler/RabbitMqPersistenceHandlerTest.php

```diff
@@ -290,10 +314,19 @@ public function testPersistRetrieveValidDataFromPdfFile()
 
         $persistenceHandler = new RabbitMqPersistenceHandler($queue, $pdfParser);
 
-        $this->assertNull($persistenceHandler->persist($resource));
+        $crawlJob = $this
+            ->getMockBuilder('Simgroep\ConcurrentSpiderBundle\CrawlJob')
+            ->setMethods(null)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->assertNull($persistenceHandler->persist($resource, $crawlJob));
     }
 
-    public function testPersistRetrieveValidDataFromWebPageWithAllDefaultValuesInFields()
+    /**
+     * @test
+     */
+    public function persistRetrieveValidDataFromWebPageWithAllDefaultValuesInFields()
```

Comment:
change name of method to begin with test that everything is consistent, and things without test on beginning is internal used for phpunittests